### DISCUSSION TITLE
News feed(s)

### DISCUSSION THREAD

**Discussion by sadortun** [GoPlan-Finance/goplan-app#23] [Ideas]
- [ ] Have a general stock market related news feed
- [ ] Have a per-stock news feed
- [ ] Analyse sentiment about a stock news with [natural languague analysis](https://cloud.google.com/natural-language#natural-language-api-demo)

**Comment by mkitzmann** [comment]
Sounds like a good idea. We should probably start some kind of roadmap to track the features we want to implement.
